SimplenoteUITests/NoteList.swift

```diff
@@ -274,4 +296,8 @@ class NoteListAssert {
         print(">>> Actual search string is '\(actualSearchString)'")
         XCTAssertEqual(actualSearchString, searchString)
     }
+
+    class func isDeselectAllButtonDisplayed() {
```

Comment:
Yeah, guess this is more of a habit for me, I'm used to starting methods that assert with `is...` ,`are...`,`has...` so that just by looking at the first few letters we know what it is going to do. In this case since it's structured in a way that all are under the `...Assert` it makes sense to not use it, will update this.